src/test/java/uk/gov/ida/eventemitter/LocalEncryptionTest.java

```diff
@@ -0,0 +1,77 @@
+package uk.gov.ida.eventemitter;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.datatype.joda.JodaModule;
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+import java.security.InvalidAlgorithmParameterException;
+import java.security.InvalidKeyException;
+import java.util.UUID;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+public class LocalEncryptionTest {
+
+    @Rule
+    public ExpectedException expectedException = ExpectedException.none();
+
+    private static final String KEY = "aesEncryptionKey";
+    private static final String INIT_VEC = "encryptionIntVec";
+    private Encrypter encrypter;
+    private Decrypter<TestEvent> decrypter;
+    private static final UUID EVENT_ID = UUID.randomUUID();
+    private static final Event EVENT = new TestEvent(EVENT_ID, DateTime.now(DateTimeZone.UTC), "eventType");
+    private ObjectMapper mapper = new ObjectMapper();
+
+    @Before
+    public void setUp() {
+        mapper.registerModule(new JodaModule());
+        encrypter = new LocalEncrypter(KEY, INIT_VEC, mapper);
+        decrypter = new LocalDecrypter(KEY, INIT_VEC, mapper);
+    }
+
+    @Test
+    public void shouldEncrypt() throws Exception {
+        final String encryptedEvent = encrypter.encrypt(EVENT);
+        final TestEvent actualEvent = decrypter.decrypt(encryptedEvent, TestEvent.class);
+
+        assertThat(actualEvent).isEqualToComparingFieldByFieldRecursively(EVENT);
+    }
+
+    @Test
+    public void ShouldLogErrorAfterFailingToEncrypt() throws Exception {
```

Comment:
Test name talks about logs, but we never actually assert anything gets logged - should we?